app/src/main/res/layout/item_custom_incoming_voice_message.xml

```diff
@@ -93,7 +93,7 @@
 
             <SeekBar
                 android:id="@+id/seekbar"
-                android:layout_width="match_parent"
+                android:layout_width="0dp"
```

Comment:
Not sure this belongs in this PR?

Reply:
it's from the commit "fix lint warnings". it's not related to this PR, but otherwise the checks in this PR will fail. But yeah i could fix it in another PR..